packages/flagship/src/lib/link.ts

```diff
@@ -27,7 +27,8 @@ export async function link(configuration: Config): Promise<boolean | object> {
   // Spawn react-native link
   const spawned = spawn('react-native', ['link'], {
     cwd: path.project.path(),
-    shell: os.win
+    shell: os.win,
+    stream: 'stderr'
```

Comment:
This will surface init errors but if everything is working correctly it won't capture the output from codepush because that's in stdout.